TranslatorReasonerAPI.yaml

```diff
@@ -422,3 +449,199 @@ components:
+          description: >-
+            Attributes associated with this node.
+      required:
+        - id_prefixes
+      additionalProperties: false
+    KMPredicate:
+      type: object
+      description: >-
+        Predicate describing relationship between a subject and an object.
+      properties:
+        subject:
+          type: string
+          example: ChemicalSubstance
+        predicate:
+          type: string
+          example: affects
+        object:
+          type: string
+          example: Protein
+        relations: 
+          type: array
+          description: >-
+            Low-level relations from the underlying source.
+          items:
+            type: string
+          example: [inhibits, activates]
+        count:
+          type: integer
+          description: >-
+            Total number of edge instances known to this knowledge source,
+            -1 if the number is not known.
+        count_by_source:
+          type: object
+          description: >-
+            Number of edge instances by their source
+          additionalProperties:
+            type: integer
+          example:
+            ChEMBL: 1234
+        attributes:
+          type: array
+          items:
+            $ref: '#/components/schemas/KMAttribute'
+          description: >-
+            Attributes associated with this predicate.
+      required:
+      - subject
+      - predicate
+      - object
+      additionalProperties: false
+    KMAttribute:
+      type: object
+      description: >-
+        Description of attribute types provided by this knowledge source.
+      properties:
+        type:
+          type: string
+          description: >-
+            CURIE of the semantic type of the attribute, from the EDAM ontology
+            if possible. If a suitable identifier does not exist, enter a
+            descriptive phrase here and submit the new type for consideration
+            by the appropriate authority.
+          example: EDAM:data_1187
+        source:
+          type: string
+          description: Source of the attribute, as a CURIE prefix.
+          example: UniProtKB
+        names:
+          type: array
+          description: >-
+            Human-readable names or labels for the attribute for attributes of 
+            given type.
+          items:
+            type: string
+          example: [PubMed Identifier]
+      required:
+        - type
+      additionalProperties: false
+    Info:
+      type: object
+      properties:
+        title:
+          type: string
+          description: >-
+            Title od this knowledge source
+          example: best ARA ever
+        description:
+          type: string
+          description: >-
+            Description od this knowledge source
+          example: This is the best ARA ever.
+        version:
+          type: string
+          description: >-
+            Version od this knowledge source
+          example: 1.0.1
+        TRAPI:
```

Comment:
For the TRAPI classes that have `additionalProperties: true` (Query, KnowledgeGraph, QueryGraph, QNode, and QEdge), would those additional properties be described in the metadata here as well?

Reply:
This proposal only addresses the structure of the knowledge graph.